src/screens/crew6/engineeringScreen.cpp

```diff
@@ -59,6 +59,8 @@ EngineeringScreen::EngineeringScreen(GuiContainer* owner, ECrewPosition crew_pos
         info.button->setSize(300, GuiElement::GuiSizeMax);
         info.damage_bar = new GuiProgressbar(info.layout, id + "_DAMAGE", 0.0, 1.0, 0.0);
         info.damage_bar->setSize(150, GuiElement::GuiSizeMax);
+        info.health_max_bar = new GuiProgressbar(info.damage_bar, id + "_MAX", 1.0, 0.0, 0.0);
+        info.health_max_bar->setSize(GuiElement::GuiSizeMax, GuiElement::GuiSizeMax);
```

Comment:
Not sure if I like this to show up for everyone all the time for a specific LARP/Scenario feature.

Reply:
I misspoke, I didn't think about only one LARP or a specific scenario, but rather the lot of scenarios when GM is active (in opposition with arcade scenarios without GM) or if the script takes into account this feature.

For example, in the scenario 56_carrierTurret, the health of the system is 'hacked' to simulate a failure, but the GM cannot tweak it and players only see the problem with the log :  
`tractorShip:setSystemHealth("warp",-.5):setSystemHealth("jumpdrive",-.5):orderStandGround():setCallSign(strikeShipNames[math.random(1,#strikeShipNames)])
			for pidx=1,8 do
				p = getPlayerShip(pidx)
				if p ~= nil and p:isValid() then
					p:addToShipLog(string.format("[%s] Help requested: Our engines are damaged beyond our ability to repair. We are located in %s",tractorShip:getCallSign(),tractorShip:getSectorName()),"#556b2f")
				end
			end`

Some examples of uses in script : 
- Max health can decrease once health is below 0 and playerShip has to be repair in a specific station
- Max health can be limited into some zones or near specific objects
- Max health can be broken during a WARP or JUMP to simulate specific failure 

Note that if you don't use it, players will not see anything.